### PR TITLE
LPS-125150 Verify app scope before include

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:app-builder

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey @naraandrade. Can you please run the `gw baseline` into the app-builder-api module? It is required because you have moved the class to there and this requires a bump in the package version.

Thx

File: modules/apps/app-builder/app-builder-api/src/main/java/com/liferay/app/builder/constants/AppBuilderWebKeys.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.app.builder.web.internal.constants;
+package com.liferay.app.builder.constants;
```
Comment (rafaprax) [inline modules/apps/app-builder/app-builder-api/src/main/java/com/liferay/app/builder/constants/AppBuilderWebKeys.java:15]: Hey @naraandrade , this change requires a version increment. Please run `gw baseline`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/apps/app-builder/app-builder-api/src/main/java/com/liferay/app/builder/constants/AppBuilderWebKeys.java (hunk 1 above)
Comment (naraandrade) [reply modules/apps/app-builder/app-builder-api/src/main/java/com/liferay/app/builder/constants/AppBuilderWebKeys.java:15]: I'll do it. Thank you :)

**Comment by naraandrade** [comment]
ci:forward

**Review by rafaprax — APPROVED** [review]

**Pull request closed without merging**